contracts/contracts/poolBooster/PoolBoosterFactoryMerkl.sol

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import { PoolBoosterMerkl } from "./PoolBoosterMerkl.sol";
+import { AbstractPoolBoosterFactory, IPoolBoostCentralRegistry } from "./AbstractPoolBoosterFactory.sol";
+
+/**
+ * @title Pool booster factory for creating Merkl pool boosters.
+ * @author Origin Protocol Inc
+ */
+contract PoolBoosterFactoryMerkl is AbstractPoolBoosterFactory {
+    uint256 public constant version = 1;
+
+    /// @notice address of the Merkl distributor
+    address public merklDistributor;
+    /// @notice address of the Merkl hash to sign
+    bytes32 public merklHashToSign;
```

Comment:
Since we don't sign this hash, I think we can rename it to just `merklHash`. That would remove some of the signing confusion.